ani-cli

```diff
@@ -316,7 +339,20 @@ while [ $# -gt 0 ]; do
             index="$2"
             shift
             ;;
+        -Se | --season)
+            [ $# -lt 2 ] && die "missing argument!"
+            seasonnum="$2"
+            shift
+            ;;        
+        
         -c | --continue) search=history ;;
+        
+        --j | --jellyfin)
+            [ $# -lt 2 ] && die "missing argument!"
```

Comment:
As noted previously, presence means true, absence means false.
Also it's `-j` not `--j` if at all